README.md

```diff
@@ -97,27 +97,43 @@ The options that we support for calculation are as follows:
 To run the globally installed CLI (see above), use the global `fqm-execution command`
 
 ```bash
-Usage: fqm-execution [options]
+Usage: fqm-execution [command] [options]
+
+Commands:
+ detailed (default)
+ raw
+ reports
+ gaps
+ dataRequirements
+ queryInfo
 
 Options:
-  -d, --debug                                 enable debug output (default: false)
-  -o, --output-type <type>                    type of output, "raw", "detailed", "reports", "gaps", "dataRequirements", "queryInfo" (default: "detailed")
-  -r, --report-type <report-type>             type of report, "individual", "summary", "subject-list"
+  --debug                                     enable debug output (default: false)
+  --report-type <report-type>                 type of report, "individual", "summary", "subject-list"
   -m, --measure-bundle <measure-bundle>       path to measure bundle
-  -p, --patient-bundles <patient-bundles...>  paths to patient bundles. Required unless output type is dataRequirements
-  -f, --as-patient-source                     Load bundles by creating cql-exec-fhir PatientSource to pass into library calls
+  -p, --patient-bundles <patient-bundles...>  paths to patient bundles. Required unless output type is dataRequirements. Note: cannot be used with --patient-ids.
+  --patient-ids <ids...>                      (with --fhir-server-url) A list of patient ids an AsyncPatientSource will use to query a fhir server for patient data. Note: cannot be used with --patient-bundles.
```

Comment:
Would it be hard to make this a list of "subject" ids and allow either Patient/id or Group/id?  If Group/id, would need to get the Patients from the same fhir-server.

Reply:
@p9g Right now we have all of the fhir server requests handled in cql-exec-fhir. As far as I can tell, it would require a small rework there as well as some changes to fqm-execution to distinguish between Group and Patient Ids. I believe it would be doable without too much effort though. Is this something you're interested in seeing?

Reply:
Added task in the backlog for this